opencensus_python_extensions_azure/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = '0.0.1'
```

Comment:
The namespace should be something like "opencensus/azure/extensions/__init__.py".

Reply:
And then functions extension would exist like "opencensus/azure/extensions/functions/init.py"